lib/vdsm/api/vdsm-api.yml

```diff
@@ -1190,6 +1190,13 @@ types:
             description: The volume path to be used in the VM XML
             name: path
             type: string
+        -   defaultvalue: null
+            description: The volume path to be used for running VMs,
```

Comment:
I'd explain how it's related to `path` property - saying this will be used for running VMs, while `path` descriptions says it's used in VM XML is IMHO confusing. Or probably just update description for `path` property?

Reply:
yes, I'll update `path`'s description

Reply:
Can we deprecate "path"? Idealy we have only one path reported, and it is uniform
for all kinds of disks.

I'm also not sure about "run_path", this was the first thing I thought about when
proposing this change, but sounds bit dirty.

Alternative terms:
- disk_path
- volume_path

Reply:
yes, I thought about deprecating it, we still use it in copy though, where it doesn't matter what the path is